internal/tools/uvmboot/main.go

```diff
@@ -308,25 +308,24 @@ func execViaGcs(cs *hcs.System, c *cli.Context) error {
 		return err
 	}
 	defer p.Close()
-	_, pout, perr, err := p.Stdio()
-	if err != nil {
-		return err
-	}
+	_, pout, perr := p.Stdio()
 	ch := make(chan error)
 	n := 0
-	asyncCopy := func(w io.Writer, r io.ReadCloser) {
+	asyncCopy := func(w io.Writer, r io.Reader, name string) {
 		n++
```

Comment:
This might be the weirdest pattern I have seen. I think this is why `error groups` were created. I know you didn't write this though.

Reply:
I think I did write this. I didn't know about error groups.

Reply:
They are slick: https://godoc.org/golang.org/x/sync/errgroup

Reply:
Cool. I'll fix this up separately later.

Reply:
Np